### DISCUSSION TITLE
清华大学 - Scholar Compass 高校文档

### DISCUSSION THREAD

**Discussion by giscus** [Scholar-Compass/content#24] [General]
# 清华大学 - Scholar Compass 高校文档

帮助高三毕业生更好地择校选专业。

https://scholar-compass.github.io/content/university/%E6%B8%85%E5%8D%8E%E5%A4%A7%E5%AD%A6/

**Comment by YDX-2147483647** [comment]
[清华常用信息/服务汇总 - THU Services](https://thu.services/)
